drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -307,34 +307,70 @@
 
 
 /* Font Awesome icons have different width - margin-right tries to compensate it */
-.xblock--drag-and-drop .feedback p:before {
+.ltr .xblock--drag-and-drop .feedback p:before,
```

Comment:
@cahrens unfortunately this is css not sass. Otherwise, could import bi-app sass.